ui/app/templates/vault/cluster/secrets/backend/overview.hbs

```diff
@@ -33,26 +33,39 @@
         <OverviewCard
           @cardTitle="Connections"
           @subText="The total number of connections to external databases that you have access to."
-          @actionText="Configure new"
-          @actionTo="vault.cluster.secrets.backend.create-root"
-          @actionQuery={{hash itemType="connection"}}
         >
-          <Hds::Text::Display class="has-top-padding-m" @tag="h2" @size="500">
-            {{format-number (if (eq this.model.connections 404) 0 this.model.connections.length)}}
-          </Hds::Text::Display>
+          <:action>
+            <Hds::Link::Standalone
+              @text="Configure new"
+              @route="vault.cluster.secrets.backend.create-root"
+              @query={{hash itemType="connection"}}
+              @icon="plus"
```

Comment:
all of these were `chevron-right` previously, I changed those related to create actions to `plus` which follows design system guidelines
<img width="1102" alt="Screenshot 2024-07-15 at 4 31 47 PM" src="https://github.com/user-attachments/assets/8b14d05c-c99a-4a43-99a2-cc445c997eb1">